variables.container.instance.tf

```diff
@@ -0,0 +1,60 @@
+variable "container_instance_count" {
+  type        = number
+  description = "The number of container instances to create"
+  default     = 2
+}
+
+variable "container_instance_name_prefix" {
+  type        = string
+  description = "The name prefix of the container instance"
+  default     = null
+}
+
+variable "container_instance_container_name" {
+  type        = string
+  description = "The name of the container instance"
+  default     = null
+}
+
+variable "container_instance_environment_variables" {
+  type = set(object({
+    name  = string
+    value = string
+  }))
+  default     = []
+  description = "List of additional environment variables to pass to the container."
+}
+
+variable "container_instance_sensitive_environment_variables" {
+  type = set(object({
+    name  = string
```

Comment:
Same, could this be a map(string)?

Reply:
As above, there could be an edge case where the key needs to be supplied from an output from another resource.

Reply:
Agreed